.github/workflows/js.yml

```diff
@@ -1,14 +1,28 @@
 name: JS tests
 on: [push, pull_request]
 jobs:
-  js-linter:
-    name: JS linter
+  js-task:
+    name: lint js dependencies
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2.0.0
-      - uses: actions/setup-node@v1
+      - name: lint js dependencies
+        uses: PoulainMaxime/github-action-lint-js/12@master
         with:
-          node-version: '13.x'
-      - run: yarn --cwd _dev/ install
-      - run: yarn --cwd _dev/ lint
+          cmd: yarn
+          path: ./_dev
+      - name: build js dependencies
```

Comment:
Actually, from this step we could create a new workflow (`build.yml`) responsible in downloading / compiling dependencies, cleaning then build / uploading the artifact.

If possible, make sure to start it **after** the tests have been run successfully.